### DISCUSSION TITLE
vdh became blank

### DISCUSSION THREAD

**Discussion by JMFRO** [aclap-dev/video-downloadhelper#818] [Help and Q&A]
### Your question

I have the 9.0.2.3 version of VDH. The icon became white and empty (see attached)
![Capture d’écran 2024-06-03 à 17 55 22](https://github.com/aclap-dev/video-downloadhelper/assets/171658628/3aa96793-d933-4c80-9146-6c737f7b55f4)
. When I read a video (I almost only use videos with password) the system says it sees no media.

### Addon & CoApp version and details

9.0.2.3

**Comment by mmsabbott** [comment]
Since the update, my icon is blank and when I click on it, it gives me an empty box. Version 9.0.2.3

**Comment by JMFRO** [comment]
answer?

Jean-Michel Frodon
Slate.fr: www.slate.fr/source/jean-michel-frodon
Projection publique: https://projection-publique.com/
Professeur associé à Sciences-Po Paris
Membre du comité pédagogique de SPEAP
http://blogs.sciences-po.fr/speap/
Honorary Professor, University of Saint Andrews (Scotland)
Président de l'association PEROU <https://www.perou-paris.org/>
